wp-content/themes/core/components/card/card-horizontal.php

```diff
@@ -0,0 +1,47 @@
+<?php
+declare( strict_types=1 );
+
+use \Tribe\Project\Templates\Components\card\Card_Controller;
+
+/*
+add data-js to allow for a fully "linked" card, like we used to do:
+
+https://inclusive-components.design/cards/
+
+d.bind_events = function() {
+	d.$el.body.on("click", ".use-target", function() {
+		a.location = c(this).find(".is-target").attr("href")
+	}).on(d.state.click, ".save-target", function(a) {
+		a.stopPropagation()
+	}),
+};
+card.style.cursor = 'pointer';
+also scope stanford 125
+*/
+
+/**
+ * @var array $args Arguments passed to the template
+ */
+// phpcs:ignore VariableAnalysis.CodeAnalysis.VariableAnalysis.UndefinedVariable
+$c = Card_Controller::factory( $args );
+?>
+
+<<?php echo $c->get_tag(); ?> <?php echo $c->get_horizontal_classes(); ?> <?php echo$c->get_attrs(); ?>>
```

Comment:
Curious if it's really necessary to have a separate template file for this card style? It looks like it just omits the description? If that's the case, I'd almost rather see us support the description in this card style too. Then in a project, if description is unused, it just doesn't render.  (fwiw, the designs [show that the larger horizontal card](the designs show that the larger horizontal card can show a description.) can show a description.)